src/openrct2/ride/RideRatings.cpp

```diff
@@ -153,17 +151,18 @@ static void RideRatingsApplyRequirementStations(RatingTuple& ratings, const Ride
 static void RideRatingsApplyRequirementSplashdown(RatingTuple& ratings, const Ride& ride, RatingsModifier modifier);
 static void RideRatingsApplyPenaltyLateralGs(RatingTuple& ratings, const Ride& ride, RatingsModifier modifier);
 
-RideRatingUpdateStates& RideRatingGetUpdateStates()
+RideRatingUpdates& RideRatingGetUpdateStates()
```

Comment:
I think we should remove this function and use `gameState.RideRatingUpdateStates` directly. I’d like @ZehMatt ’s opinion first, though.

Reply:
I think so too, at the moment its exposed so S4/S6/Park can access it but we now pass the game state to be imported/exported rather than relying on a global so its better to remove it.